### ISSUE TITLE
NTR: CT, MRI or angiography: paraspinal region AVM (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8977]
New term request CT, MRI or angiography: paraspinal region AVM (elthat_130802104918): Arteriovenous malformation  UMLS: 5232741

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000592
- label: Paraspinal arteriovenous malformation
- parents: [Arteriovenous malformation (HP:0100026)](https://hpo.jax.org/app/browse/term/HP:0100026)
- pmid: 
- def: An anomalous configuration of blood vessels that shunts arterial blood directly into veins without passing through the capillaries and that is located in adjacent to the spine, i.e., in a location that is normally occupied by muscles, ligaments, nerves, and other tissues that run alongside the spinal column.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8977
- synonyms:
